Dockerfile.dapper

```diff
@@ -21,6 +21,9 @@ ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 \
 
 RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
 
+ENV RKE_URL_amd64=https://github.com/rancher/rke/releases/download/v0.1.10-rc1/rke_linux-amd64
```

Comment:
we should change it to the official release once it's up

Reply:
sure